skbio/stats/tests/test_composition.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import, division, print_function
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+
+import numpy as np
+from numpy.testing import assert_array_almost_equal
```

Comment:
Thanks for chiming in! Actually, `npt` is straight from [our docs](http://scikit-bio.org/docs/latest/development/coding_guidelines.html#how-do-i-organize-my-modules-source-files) but it's kind of hidden :)